feathr-example/README.md

```diff
@@ -1 +1,73 @@
-TBD
+# Feathr Home credit default notebooks
+
+These notebooks are conversions from the `feast` implementation to `feathr` implemenation - the `feast` implmenation was created by our client as an exercise to explore `feast` capabilities. 
```

Comment:
Nit: typo on "feathr `implemenation`" and I think the word implementation might not be needed after `feathr`. We can also remove the sentence mentioning the `customer` (the second part of the sentence altogether.)

Reply:
Changed the wording. Thanks.